runtime/commonMain/src/kotlinx/serialization/SerialDescriptor.kt

```diff
@@ -117,7 +117,7 @@ package kotlinx.serialization
  * }
  * ```
  *
- * For a classes that are represented as a single primitive value, [PrimitiveDescriptor] builder function can be used instead.
+ * For classes representing a single primitive value, [PrimitiveDescriptor] builder function can be used instead.
```

Comment:
I'd rather stick with a previous wording --- a value represents a class, not vice versa. Class can be serialized to many different forms of values with different serializers

Reply:
Sure, don't mind